README.md

```diff
@@ -64,10 +64,66 @@ tags:
   kubernetes_label_ServiceCode: <string>
 ```
 
-## Update below resource documentation later 
+### Environment specific Default values 
+
+Below values are set in flux repositories and all ASO resources will use these values internally. 
+
+for e.g. NameSpaceQueues will get created inside `serviceBusNamespaceName` namaspace and postgres database will get created inside `postgresServerName` server.
```

Comment:
Can we rename the file. i think it should be _namespace-queue.yaml for easy readability. Even the documentation should refer to them as Namespace Queues

Reply:
@kbabigumira : Done